gradle-plugin/README.md

```diff
@@ -0,0 +1,59 @@
+# Contrast Gradle Plugin
+
+Gradle plugin for including the Contrast Security analysis in Java web applications 
+
+## Building
+Requires Java 21 to build
+
+Use `./gradlew build` to build the plugin
+
+## Publishing to MavenLocal
```

Comment:
We should look into getting this released on `https://plugins.gradle.org/` as I think making people pull this down and install it is not the correct approach.

Reply:
yeah the actual release and publishing work is part of separate tickets. I should have put this under the development section because this was just meant for local testing while it's still in development. Might not even be necessary to mention here